src/Analyzer.PowerShellRuleEngine/PowerShellRuleEvaluation.cs

```diff
@@ -26,7 +29,7 @@ public class PowerShellRuleEvaluation : IEvaluation
         public string HelpUri { get; }
 
         /// <inheritdoc/>
-        public Severity Severity { get; } = Severity.Medium;
```

Comment:
This was because the default is medium and is not a necessary parameter

Reply:
Is this a question? That was there because TTK didn't have severities so we were using Medium for everything, now the severity is always set in the constructor